File: Java/JavaApplication/app/src/main/java/com/example/android/javaapplication/MainActivity.java
package com.example.android.javaapplication;


import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    private Button button_minus;
    private TextView quantity_text_view;
    private Button button_plus;
    private TextView price_text_view;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        quantity_text_view = findViewById(R.id.quantity_text_view);
        price_text_view = findViewById(R.id.price_text_view);

        button_minus = findViewById(R.id.button_minus);
        button_minus.setOnClickListener(v -> {
            int x = Integer.parseInt(quantity_text_view.getText().toString());
            if (x <= 0)
                quantity_text_view.setText(String.valueOf(x));
            else {
                x--;
                quantity_text_view.setText(String.valueOf(x));
            }
        });

        button_plus = findViewById(R.id.button_plus);
        button_plus.setOnClickListener(v -> {
            int x = Integer.parseInt(quantity_text_view.getText().toString());
            x++;
            quantity_text_view.setText(String.valueOf(x));

        });

    }

    public void click_order_button(View view) {
            int x = Integer.parseInt(quantity_text_view.getText().toString());
            price_text_view.setText(String.valueOf(x*5));
    }
}

File: Kotlin/MyApplication/app/src/main/java/com/example/android/myapplication/MainActivity.kt
package com.example.android.myapplication

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.TextView
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private lateinit var textView: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        textView=findViewById(R.id.textView)
    }

    fun toastMy (view:View){
        val myToast = Toast.makeText(this,"Привет",Toast.LENGTH_SHORT)
        myToast.show()
    }

    fun myCount (view:View){
        var count = textView.text.toString().toInt()
        count++
        textView.text=count.toString()
    }

    fun randomMy(view: View){
        val intentRandom = Intent(this,SecondActivity::class.java)
        startActivity(intentRandom)
    }
}